src/models/continuity/ContGateTurn.ts

```diff
@@ -17,7 +18,12 @@ export default class ContGateTurn
 
   duration = 8;
 
-  centerPoints: [number, number][] = [];
+  centerPoints: Map<number, [number, number]> = new Map<
+    number,
+    [number, number]
+  >();
```

Comment:
Good idea! Have you considered other options? What are the pros and cons?

Reply:
- Map of each dot -> their center dot. Or each dot -> reference point. Look into CalChart3 and Kevin's write-up about reference points (big benefit is not having to worry if there is a marcher at that spot). In CalChart3, the reference points were in the Dot data structure itself?
  - Note that in Step2, it uses distance from a reference point to figure out the sub groups
- Numbering each dot (think of follow the leader style)

Gate turn
```
[
  { centerPoint: [2, 2], dotIds: [id1, id2]
  ...
]
```

Follow the leader (need to look into performance)
```
{
  ids: [id1, id2, id3],
  path: [[2,2], [2,3], ...] // or continuities
}
```

2 step
```
startPoint: [2, 2],
endPoint: [6, 2],
```

What does the UX look like?
- Select all dots in a gate turn group. Then, choose the center point.

What does importing from CalChart3 look like?
- Import directly from `.shw` file
- Import from a viewpsheet file
- Do not do a 1 to 1 translation of complex continuities

Step 2 turns into 2 step drill where the UI guides the user to choose where the step 2 drill starts and ends

Where does this information get stored?
- In `StuntSheetDot`
- In `ContGateTurn`

Will iron out and finalize in this issue: https://github.com/calband/calchart-redesign/issues/128